extensions/podman/src/podman-cleanup-windows.ts

```diff
@@ -0,0 +1,75 @@
+import type { ProviderCleanupExecuteOptions } from '@podman-desktop/api';
+import { process } from '@podman-desktop/api';
+
+import { homedir } from 'node:os';
+import { resolve as pathResolve } from 'node:path';
+import { AbsPodmanCleanup } from './podman-cleanup-abstract';
+
+export class PodmanCleanupWindows extends AbsPodmanCleanup {
+  async stopPodmanProcesses(options: ProviderCleanupExecuteOptions): Promise<void> {
+    // stop all running podman machine if there are some
```

Comment:
You should kill win-sshproxy.exe as it owns the pipes and I noticed that after I did cleanup, Docker compatibilty does not work anymore

Reply:
added kill of the processes on Windows side